doc/ref/grphomom.xml

```diff
@@ -391,7 +391,7 @@ to form groups of automorphisms.
 
 Usually the best way to calculate in a group of automorphisms is to
 translate all calculations to an isomorphic group in a representation, for
```

Comment:
OK, unrelated to this PR, but that comma really stumped me, and makes no sense to me. But perhaps native speakers like @wilfwilson or @ChrisJefferson might disagree?
```suggestion
translate all calculations to an isomorphic group in a representation for
```

Reply:
Agree that the comma should go.